src/constants.js

```diff
@@ -59,6 +59,9 @@ export const CONFIG_DIRECTORY = getDirectory('config');
 export const LINK_REGISTRY_DIRECTORY = path.join(CONFIG_DIRECTORY, 'link');
 export const GLOBAL_MODULE_DIRECTORY = path.join(CONFIG_DIRECTORY, 'global');
 
+export const POSIX_GLOBAL_PREFIX = '/usr/local';
```

Comment:
Perhaps we should try both. `/usr/local` is good if it's writable by the user, otherwise the directory that Node.js is installed from (I think there's some property on `process` that has the full path of the Node.js process?)